packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -1277,9 +1285,15 @@ export default abstract class SqlIntegration
     const { rows, statistics } = await this.runQuery(query, setExternalId);
     return {
       rows: rows.map((row) => {
+        const dimensionData: Record<string, string> = {};
+        Object.entries(row)
+          .filter(([key, _]) => key.startsWith("dim"))
```

Comment:
question: The filter above uses `dim_` but this one uses just `dim`. Should we update them to match?